test/LexerTest.php

```diff
@@ -39,126 +39,176 @@ public function testLexer($rql, $expected)
     public function lexerProvider()
     {
         return array(
-            'complex eq search concatenated fields' => array('eq(name-part+test,12)', array(
```

Comment:
Blaming my IDE too early I did. The testcases to the beginning of the array I added